supervisor/misc/tasks.py

```diff
@@ -195,16 +189,30 @@ async def _watchdog_homeassistant_api(self):
             _LOGGER.warning("Watchdog missed an Home Assistant Core API response.")
             return
 
-        _LOGGER.error(
-            "Watchdog missed %s Home Assistant Core API responses in a row. Restarting Home Assistant Core API!",
-            HASS_WATCHDOG_MAX_API_ATTEMPTS,
-        )
+        if safe_mode := reanimate_fails == HASS_WATCHDOG_MAX_REANIMATE_ATTEMPTS:
+            _LOGGER.critical(
+                "Watchdog cannot reanimate Home Assistant Core, failed all %s attempts. Restarting into safe mode",
+                reanimate_fails,
+            )
+        else:
+            _LOGGER.error(
+                "Watchdog missed %s Home Assistant Core API responses in a row. Restarting Home Assistant Core API!",
+                HASS_WATCHDOG_MAX_API_ATTEMPTS,
+            )
```

Comment:
E.g. like this. This also readds the previous comment, which seems to got lost in the rebase.

```suggestion
        if reanimate_fails < HASS_WATCHDOG_MAX_REANIMATE_ATTEMPTS:
            safe_mode = False
            _LOGGER.error(
                "Watchdog missed %s Home Assistant Core API responses in a row. Restarting Home Assistant Core!",
                HASS_WATCHDOG_MAX_API_ATTEMPTS,
            )
        elif reanimate_fails == HASS_WATCHDOG_MAX_REANIMATE_ATTEMPTS:
            safe_mode = False
            _LOGGER.critical(
                "Watchdog cannot reanimate Home Assistant Core, failed all %s attempts. Restarting into safe mode",
                reanimate_fails,
            )
        else:
            return
```

Reply:
I did it a bit differently but everything around max reanimate attempts is together now